corehq/apps/reports/standard/deployments.py

```diff
@@ -309,13 +320,19 @@ def process_rows(self, users, fmt_for_export=False):
                 build_version = last_build.get('build_version') or build_version
                 if last_build.get('app_id'):
                     app_name = self.get_app_name(last_build['app_id'])
+                last_build_profile_id = last_build.get('build_profile_id')
+                if last_build_profile_id:
+                    last_build_profile_name = _("Unknown")
+                    build_profiles = self._get_app_details(last_build['app_id']).get('build_profiles', {})
+                    if last_build_profile_id in build_profiles:
+                        last_build_profile_name = build_profiles[last_build_profile_id].name
```

Comment:
This could be simplified:
```python
last_build_profile_id = last_build.get('build_profile_id')
build_profiles = self._get_app_details(last_build['app_id']).get('build_profiles', {})
last_build_profile_name = build_profiles.get(last_build_profile_id, _('Unknown))
```

Reply:
Isn't it better to have earlier exits if things are missing?
If this was one get statement I would not mind it, but its a series of fallbacks.

Reply:
I think it's just a style thing - either is fine